src/textStyles.ts

```diff
@@ -1,113 +1,134 @@
+export function getFontWeight(type?: FontWeight) {
+  switch (type) {
+    case 'light':
+      return `font-weight: 300;`;
+    case 'regular':
+      return `font-weight: 400;`;
+    case 'medium':
+      return `font-weight: 500;`;
+    case 'bold':
+      return `font-weight: 700;`;
+    default:
+      return '';
+  }
 }
 
-export const TextStylings = {
-  [TextStyles.giga]: {
-    fontSize: remCalc('76px'),
-    lineHeight: remCalc('95px'),
-    fontWeight: 300,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.mega]: {
-    fontSize: remCalc('54px'),
-    lineHeight: remCalc('68px'),
-    fontWeight: 300,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.alpha]: {
-    fontSize: remCalc('48px'),
-    lineHeight: remCalc('60px'),
-    fontWeight: 300,
-    letterSpacing: '0.87px',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.beta]: {
-    fontSize: remCalc('28px'),
-    lineHeight: remCalc('35px'),
-    fontWeight: 300,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.gamma]: {
-    fontSize: remCalc('22px'),
-    lineHeight: remCalc('28px'),
-    fontWeight: 700,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.upperCase,
-  },
-  [TextStyles.delta]: {
-    fontSize: remCalc('18px'),
-    lineHeight: remCalc('23px'),
-    fontWeight: 500,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.epsilon]: {
-    fontSize: remCalc('16px'),
-    lineHeight: remCalc('20px'),
-    fontWeight: 500,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.zeta]: {
-    fontSize: remCalc('14px'),
-    lineHeight: remCalc('18px'),
-    fontWeight: 300,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.omega]: {
-    fontSize: remCalc('14px'),
-    lineHeight: remCalc('18px'),
-    fontWeight: 700,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.caption]: {
-    fontSize: remCalc('12px'),
-    lineHeight: remCalc('18px'),
-    fontWeight: 400,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.legal]: {
-    fontSize: remCalc('11px'),
-    lineHeight: remCalc('17px'),
-    fontWeight: 400,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-  [TextStyles.regular]: {
-    fontSize: remCalc('16px'),
-    lineHeight: remCalc('24px'),
-    fontWeight: 400,
-    letterSpacing: 'initial',
-    textTransform: TextTransform.none,
-  },
-};
+export interface FontStyleProps {
+  size?: FontSize;
+  weight?: FontWeight;
+  lineHeight?: LineHeight;
+}
+
+export function getFontStyle({ size, weight, lineHeight = size }: FontStyleProps) {
+  return css`
+    ${getFontSize(size)}
+    ${getFontLineHeight(lineHeight)}
+    ${getFontWeight(weight)}
+  `;
+}
+
+/**
+ * @deprecated
```

Comment:
In my opinion these deprecated things can be specified in Pharos - keeping Precise clean.